src/ViewComposers/CountryCodeViewComposer.php

```diff
@@ -2,6 +2,7 @@
 
 namespace PodPoint\I18n\ViewComposers;
 
+use GuzzleHttp\Utils;
 use Illuminate\View\View;
 use Illuminate\Config\Repository;
 use PodPoint\I18n\CountryCode;
```

Comment:
Pretty sure the standard now is to have all our imports ordered alphabetically.

Maybe some linting enfine would be nice, same thing might be OOS.